pkgs/servers/code-server/default.nix

```diff
@@ -129,6 +130,9 @@ in stdenv.mkDerivation rec {
   '';
 
   configurePhase = ''
+    # run yarn offline by default
+    echo '--install.offline true' >> .yarnrc
```

Comment:
can you use `yarn config set` here?

Reply:
As far as I can tell, this flag is only available in `.yarnrc`, not in the config -- I tried using `yarn config set` instead, but it seemed to have no effect.

Reply:
ok